src/commands/client/range/index.ts

```diff
@@ -2,14 +2,22 @@ import type { NAMESPACE, Vector } from "@commands/client/types";
 import type { Dict } from "@commands/client/types";
 import { Command } from "@commands/command";
 
-type RangeCommandPayload = {
+type BaseRangePayload = {
   cursor: number | string;
   limit: number;
   includeVectors?: boolean;
   includeMetadata?: boolean;
   includeData?: boolean;
 };
 
+type RangeCommandPayload = BaseRangePayload &
+  (
+    | {}
+    | {
+        prefix: string;
+      }
+  );
+
```

Comment:
why don't we add a simple prefix?: string to the old RangeCommandPayload?

Reply:
oh you are right, did not realize it since i did it like this in delete, fixing.